webpack/scenes/Subscriptions/SubscriptionsPage.js

```diff
@@ -59,7 +71,20 @@ class SubscriptionsPage extends Component {
   render() {
     const { tasks, subscriptions } = this.props;
     const { disconnected } = subscriptions;
-    const disableManifestActions = tasks.length > 0 || disconnected;
+    const taskInProgress = tasks.length > 0;
+    const disableManifestActions = taskInProgress || disconnected;
+
+    let task = null;
+
+    if (taskInProgress) {
+      [task] = tasks;
```

Comment:
If there's multiple tasks in progress, this ensures that only the first one is displayed, right? Do I understand it correctly?

Reply:
Yeah, and there should never be more than one of the `BLOCKING_FOREMAN_TASK_TYPES` going on anyway or something else is really wrong.